docs/user-guide/creating-application/workflow/cd-pipeline.md

```diff
@@ -26,19 +45,25 @@ Your CD pipeline will can be configured for the pre-deployment stage, the deploy
 | Deployment Strategy | Select the type of deployment strategy that you want to enable by clicking `Add Deployment Strategy` |
 | Post-deployment stage | If you need to run any configurations and provide secrets after the deployment, mention those here |
 
-![](../../../.gitbook/assets/configuring-cd-pipeline.png)
 
 ### 1. Pipeline Name
 
-Inside the Pipeline name column, give a name to your Continuous deployment.
+Pipeline name will be autogenerated.
 
 ### 2. Deploy to Environment
 
-Select the environment where you want to deploy your application. Once you select the environment, it will display the `Namespace` corresponding to your selected environment automatically.
+As we discussed above, Select the environment where you want to deploy your application. Once you select the environment, it will display the `Namespace` corresponding to your selected environment automatically.
+
+## Stages
+There are 3 dropdowns given below:
+
+* **Pre-deployment stage**
+* **Deployment stage**
+* **Post-deployment stage**
 
-### 3. Pre-deployment Stage
+### 3. Pre-deployment stage
 
-Sometimes you encounter a requirement where you have to Configure actions like DB migration, which you want to run before the deployment. The `Pre-deployment Stage` comes into the picture in such scenarios.
+Sometimes you encounter a requirement where you have to Configure actions like DB migration, which you want to run before the deployment. Then the `Pre-deployment Stage` comes into the picture in such scenarios.
```

Comment:
Sometime one has a requirement where certain actions like DB migration are to be executed before deployment, `Pre-deployment stage` should be used to configure these actions.